src/gui/plugins/component_inspector/ComponentInspector.qml

```diff
@@ -227,6 +247,99 @@ Rectangle {
     }
   }
 
+  Dialog {
+    id: addSystemDialog
+    modal: false
+    focus: true
+    title: "Add System"
+    closePolicy: Popup.CloseOnEscape
+
+    ColumnLayout {
+      GridLayout {
+        columns: 2
+        columnSpacing: 30
+        Text {
+          text: "Name"
+          Layout.row: 0
+          Layout.column: 0
+        }
+
+        TextField {
+          id: nameField
+          selectByMouse: true
+          Layout.row: 0
+          Layout.column: 1
+          Layout.minimumWidth: 250
+          onTextEdited: {
+            addSystemDialog.updateButtonState();
+          }
+        }
+
+        Text {
+          text: "Filename"
+          Layout.row: 1
+          Layout.column: 0
+        }
+
+        TextField {
+          id: filenameField
+          selectByMouse: true
+          Layout.row: 1
+          Layout.column: 1
+          Layout.minimumWidth: 250
+          onTextEdited: {
+            addSystemDialog.updateButtonState();
+          }
+        }
+      }
+
+      Text {
+        id: innerxmlLabel
+        text: "Innerxml"
+      }
+
+      Flickable {
+        id: innerxmlFlickable
+        Layout.minimumHeight: 300
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+
+        flickableDirection: Flickable.VerticalFlick
+        // contentY: 200
+        TextArea.flickable: TextArea {
+          id: innerxmlField
+          wrapMode: Text.WordWrap
+          selectByMouse: true
+          textFormat: TextEdit.PlainText
+          font.pointSize: 10
+        }
+        ScrollBar.vertical: ScrollBar {}
+      }
+    }
+
+    footer: DialogButtonBox {
+      id: buttons
+      standardButtons: Dialog.Ok | Dialog.Cancel
+    }
+
+    onOpened: {
+      buttons.standardButton(Dialog.Ok).enabled = false;
```

Comment:
After I inserted one plugin, I opened the dialog again to insert a plugin to a different model. It was nice that the dialog was already pre-populated, because I wanted to add the same plugin. But the `OK` button was disabled, so I couldn't add it.

Reply:
fixed. [da4f16a](https://github.com/gazebosim/gz-sim/pull/1549/commits/da4f16a0067dcce3bd0ae96d543786058646effc)